src/main/java/net/sf/jabref/model/groups/GroupsParser.java

```diff
@@ -1,16 +1,15 @@
-package net.sf.jabref.logic.groups;
+package net.sf.jabref.model.groups;
```

Comment:
This should be logic, I think (as also the bib database parser/writer are logic)